src/electrum/electrs.cpp

```diff
@@ -0,0 +1,115 @@
+#include "electrum/electrs.h"
+#include "util.h"
+#include "utilhttp.h"
+
+#include <map>
+#include <regex>
+#include <sstream>
+
+#include <boost/dll/runtime_symbol_info.hpp>
+#include <boost/filesystem.hpp>
+#include <boost/process.hpp>
+
+static std::string monitoring_port() { return GetArg("-electrummonitoringport", "4224"); }
+namespace electrum
+{
+std::string electrs_path()
+{
+    // look for electrs in same path as bitcoind
+    const std::string def =
+        boost::process::search_path("electrs", {boost::dll::program_location().remove_filename()}).string();
+
+    const std::string path = GetArg("-electrumexec", def);
+
+    if (!boost::filesystem::exists(path))
+    {
+        std::stringstream ss;
```

Comment:
To help the end user, let's have a different error message if the path is "".  Maybe "Electrum Server electrs was not found alongside bitcoind.  Please specify a location using -electrumexec"

Reply:
Done